src/styles/index.scss

```diff
@@ -1,3 +1,91 @@
+@import "./stars.scss";
+$--blue-color: #00acdc;
+$--main-color-text: #060b35;
+$--secondary-color-text: #616070;
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  color: $--main-color-text;
+  font-weight: 400;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+
+  &__product-image {
+    margin: 16px 3px 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-top: 0;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $--secondary-color-text;
+  }
+
+  &__rating {
+    &-reviews {
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $--secondary-color-text;
+    }
+
+    &-number {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 167px;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. just use padding